origami.json

```diff
@@ -1,7 +1,7 @@
 {
     "description": "Tools to \"brand\" other Origami components.",
     "origamiType": "module",
-    "origamiVersion": 1,
+    "origamiVersion": 2,
```

Comment:
```suggestion
    "origamiVersion": "2.0",
```

Reply:
I missed this in my previous review, apologies for that.
The version is meant to be a string which is a "2.0"